src/main/resources/application.yaml

```diff
@@ -6,7 +6,6 @@ management:
     web:
       base-path: /
```

Comment:
iirc, pipeline will start passing with these changes present:
```
management:
  endpoint:
    health:
      show-details: "always"
  endpoints:
    web:
      base-path: /
```

Reply:
I'll create another PR for pipeline changes.